bridge/test/nftbridge/NFTBridge_test.js

```diff
@@ -121,6 +122,46 @@ contract("Bridge NFT", async function(accounts) {
       assert.equal(result, "v1");
     });
 
+    it("should call the token creator", async function () {

```

Comment:
I'd maybe rename to "should return expected creator when calling getTokenCreator"

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/268/commits/9c4e1c7b64b7379ff804bb3f768eed9c96a7461b